daml_dit_ddit/subcommand_build.py

```diff
@@ -97,13 +98,30 @@ def walk_and_do(fn, src_dir):
         bytecode_compile=True,
         deterministic_timestamp=True)
 
+DAML_YAML_NAME = 'daml.yaml'
+
+def daml_yaml_version():
+    with open('daml.yaml', "r") as f:
+        daml_yaml = yaml.safe_load(f.read())
+
+        LOG.debug(f'{DAML_YAML_NAME}: %r', daml_yaml)
```

Comment:
Why f-string notation _and_ `%r`?

Reply:
Mainly to distinguish between the constant (which should be static) and `daml_yaml`, which is the dynamic and changing portion of this particular log message.

Reply:
(And whatever performance concerns occur due to the early stringization of the constant are not likely to matter at all.)